tests/benchmark.log

```diff
@@ -0,0 +1,26 @@
+  SHA-256                  :\s*\d+ Kb/s,\s*\d+ cycles/byte

+  SHA-512                  :\s*\d+ Kb/s,\s*\d+ cycles/byte

+  AES-CBC-128              :\s*\d+ Kb/s,\s*\d+ cycles/byte

+  AES-CBC-192              :\s*\d+ Kb/s,\s*\d+ cycles/byte

+  AES-CBC-256              :\s*\d+ Kb/s,\s*\d+ cycles/byte

+  AES-GCM-128              :\s*\d+ Kb/s,\s*\d+ cycles/byte

+  AES-GCM-192              :\s*\d+ Kb/s,\s*\d+ cycles/byte

+  AES-GCM-256              :\s*\d+ Kb/s,\s*\d+ cycles/byte

+  AES-CCM-128              :\s*\d+ Kb/s,\s*\d+ cycles/byte

+  AES-CCM-192              :\s*\d+ Kb/s,\s*\d+ cycles/byte

+  AES-CCM-256              :\s*\d+ Kb/s,\s*\d+ cycles/byte

+  CTR_DRBG \(NOPR\)          :\s*\d+ Kb/s,\s*\d+ cycles/byte

+  CTR_DRBG \(PR\)            :\s*\d+ Kb/s,\s*\d+ cycles/byte

+  HMAC_DRBG SHA-256 \(NOPR\) :\s*\d+ Kb/s,\s*\d+ cycles/byte

+  HMAC_DRBG SHA-256 \(PR\)   :\s*\d+ Kb/s,\s*\d+ cycles/byte

+  RSA-2048                 :\s*\d+ ms/ public

+  RSA-2048                 :\s*\d+ ms/private

+  RSA-4096                 :\s*\d+ ms/ public

+  RSA-4096                 :\s*\d+ ms/private

+  ECDHE-secp384r1          :\s*\d+ ms/handshake

+  ECDHE-secp256r1          :\s*\d+ ms/handshake

+  ECDHE-Curve25519         :\s*\d+ ms/handshake

+  ECDH-secp384r1           :\s*\d+ ms/handshake

+  ECDH-secp256r1           :\s*\d+ ms/handshake

+  ECDH-Curve25519          :\s*\d+ ms/handshake


```

Comment:
We can do that if the code is suppose to change this space. Since it is fixed space we don't need to worry about it. Idea is to keep this approach simple and stupid and only put effort in regex for complex patterns.

Also one thing to bear in mind is that htrun first tries to compare a line character by character. In this case a line with special characters will also match. But if this comparison fails then htrun tries to use the line form the template as a regex. Hence if we remove any character from the template line (i.e. treating it as a regex) then we have to be careful of the characters in the line (regex). As characters like ```()<>.*+\``` are used as special meaning chars in the regex. They would then need to be escaped.